doc/modules/cassandra/pages/developing/cql/cql_singlefile.adoc

```diff
@@ -2235,6 +2235,57 @@ LIST ROLES;
 
 but only roles with the `LOGIN` privilege are included in the output.
 
+[[databaseIdentity]]
+=== Database Identities
+
+[[AddIdentityStmt]]
+==== ADD IDENTITY
+
+_Syntax:_
+
+bc(syntax).. +
+::= ADD IDENTITY ( IF NOT EXISTS )? TO ROLE ?
+
+_Sample:_
+
+bc(sample). +
+ADD IDENTITY 'id1' TO ROLE 'role1';
+
+Only a `SUPERUSER` can add an identity
```

Comment:
Is it still true? Asking because `org.apache.cassandra.cql3.statements.CreateRoleStatement#authorize` checks for CREATE permission not SUPERUSER.